composer.json

```diff
@@ -24,32 +24,33 @@
     "source": "https://github.com/ZooRoyal/coding-standard"
   },
   "require": {
-    "php": "^7.4",
+    "php": "^8.0",
+    "composer/package-versions-deprecated": "^1.8",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
     "doctrine/annotations": "^1.10",
-    "guzzlehttp/psr7": "^1.4",
-    "knplabs/github-api": "^2.19",
+    "guzzlehttp/psr7": "^1.8 || ^2.1",
+    "knplabs/github-api": "3.4.0 as 2.20.0",
     "mindplay/composer-locator": "^2.1.4",
     "ocramius/proxy-manager": " >= 2.2.3 < 3",
     "php-di/php-di": "^6.3",
     "php-parallel-lint/php-console-highlighter": "^0.5",
     "php-parallel-lint/php-parallel-lint": "^1.2",
     "phpcompatibility/php-compatibility": "^9.3 >=9.3.5",
-    "phpmd/phpmd": "^2.9.1",
+    "phpmd/phpmd": "^2.11.0",
     "phpstan/extension-installer": "^1.1",
     "phpstan/phpstan": "^0.12.87",
     "phpstan/phpstan-doctrine": "^0.12.13",
     "phpstan/phpstan-mockery": "^0.12.12",
     "phpstan/phpstan-phpunit": "^0.12.17",
     "phpstan/phpstan-symfony": "^0.12.16",
     "sebastian/phpcpd": "^6.0.2",
-    "slevomat/coding-standard": "^6.4.1",
-    "squizlabs/php_codesniffer": "^3.5",
-    "symfony/console": "^3.4.43 || ^4.2 || ^5.0",
-    "symfony/event-dispatcher": "^4.4.13 || ^5.2",
-    "symfony/filesystem": "^3.4.43 || ^4.2 || ^5.2",
+    "slevomat/coding-standard": "^6.4.1 || ^7.0",
```

Comment:
Wenn möglich würde ich 6.x raus werfen. ich glaube es haben sich namen und parameter von sniffs in der config geändert und deshalb funktioniert die eh nur richtig für eine Version